training/nvidia-bootc/containers-storage.conf

```diff
@@ -0,0 +1,13 @@
+[storage]
+  driver = "overlay"
+[storage.options]
+    size = ""
+    remap-uids = ""
+    remap-gids = ""
+    ignore_chown_errors = ""
+    remap-user = ""
+    remap-group = ""
+    skip_mount_home = ""
+    mount_program = "/usr/bin/fuse-overlayfs"
+    mountopt = ""
+additionalimagestores = [ "/usr/lib/containers/storage",]
```

Comment:
We will need to allow non-root users access (at lest read) to this path.
Probably need to sort out selinux as well.

Reply:
The read access is set in the Containerfile. We could limit to a specific group, too.